.eslintrc.js

```diff
@@ -449,6 +449,8 @@ module.exports = {
     $ReadOnlyArray: 'readonly',
     $Shape: 'readonly',
     AnimationFrameID: 'readonly',
+    bigint: 'readonly',
```

Comment:
😮 Is this due to the type annotation?

Reply:
If I remember correctly, yes. I'll double check

Reply:
I guess there's a bunch of types in here already. Which is unfortunate since it'll cover up mistakes.

Reply:
Not sure how we could have eslint just ignore flow files. I think it's only easy by the full file name.

Good thing is that Flow will still catch `bigint` as undefined regardless of the config here, so really only non-flow files are a bit at risk. Seems okay to me.